Model/src/main/java/org/apidb/apicommon/model/report/ai/expression/DailyCostMonitor.java

```diff
@@ -0,0 +1,141 @@
+package org.apidb.apicommon.model.report.ai.expression;
+
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.Reader;
+import java.io.Writer;
+import java.nio.file.Path;
+import java.time.LocalDate;
+import java.util.Optional;
+
+import org.gusdb.fgputil.FormatUtil;
+import org.gusdb.fgputil.IoUtil;
+import org.gusdb.fgputil.cache.disk.DirectoryLock;
+import org.gusdb.wdk.model.WdkModel;
+import org.gusdb.wdk.model.WdkModelException;
+import org.gusdb.wdk.model.WdkRuntimeException;
+import org.json.JSONObject;
+
+import com.openai.models.CompletionUsage;
+
+public class DailyCostMonitor {
+
+  private static final String DAILY_COST_ACCUMULATION_FILE = "daily_cost_accumulation.txt";
+
+  // model prop keys
+  private static final String MAX_DAILY_DOLLAR_COST_PROP_NAME = "MAX_DAILY_AI_EXPRESSION_DOLLAR_COST";
+  private static final String PLAN_COST_PER_1M_INPUT_TOKENS_PROP_NAME = "COST_PER_1M_AI_INPUT_TOKENs";
```

Comment:
FYI, there's a lowercase `s` typo in the value

Reply:
These costs per million tokens are also in dollars, I assume. The person setting this in a config file would probably assume this given the name of the MAX_DAILY prop.

Reply:
Will add dollars to name to eliminate ambiguity

Reply:
And fix typo